doc/source/groupby.rst

```diff
@@ -1087,6 +1087,23 @@ To see the order in which each row appears within its group, use the
 
    df.groupby('A').cumcount(ascending=False)  # kwarg only
 
+Enumerate groups
+~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.20.0
+
+To see the ordering of the groups themselves, you can use the ``ngroup``
+method:
```

Comment:
I would add a comment differentiating this from ``.cumcount`` (when to use which)